src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
-const Home = () => {
+import React from "react";
+import { Button } from "react-bootstrap";
+
+
+const Home = props => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="d-flex flex-column justify-content-center ">
+      <section className="align-self-center text-center">
+        <h1> {"{...}"} </h1>
+        <h2> Hello Coder ! </h2>
+        <h3> Are you ready for the next challenge ?
+        <p> Take a Quiz </p>
+        </h3>
+      </section>
+      <section className="align-self-center text-center">
+        <ul className="list-unstyled">
+          <li> HTML <Button href="Quiz" > HTML</Button> </li>
```

Comment:
It looks odd to see `HTML HTML` — the user only needs to see it once, same for CSS and JavaScript. Let's remove the redundant text